mix.exs

```diff
@@ -35,7 +35,8 @@ defmodule AshPostgres.MixProject do
       aliases: aliases(),
       package: package(),
       source_url: "https://github.com/ash-project/ash_postgres",
-      homepage_url: "https://github.com/ash-project/ash_postgres"
+      homepage_url: "https://github.com/ash-project/ash_postgres",
+      consolidate_protocols: Mix.env() != :test
```

Comment:
this cleans up a warning that was appearing a lot during tests 

```
warning: the Inspect protocol has already been consolidated, an implementation for :"Elixir.Post.test creating initial snapshots the migration sets up resources correctly" has no effect. If you want to implement protocols after compilation or during tests, check the "Consolidation" section in the Protocol module documentation
```

https://hexdocs.pm/elixir/1.12.3/Protocol.html#module-consolidation

the only downside mentioned was that it may impact test suite performance but I didn't notice an impact. My changes actually shortened test time by about 30s by combining a few of migration generator tests